### ISSUE TITLE
Sometimes the changelog icon shows with no changelog. 

### ISSUE THREAD

**Issue by Pickysaurus** [Nexus-Mods/Vortex#9140]
![image](https://user-images.githubusercontent.com/31670524/115373714-0a18ee00-a1c4-11eb-8860-9a7df23ebc9d.png)

This happens sometimes and I'm not sure why. 

```
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.archiveId = "GFCYksbhQ"
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.allowRating = true
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.author = "aers - Nukem - Ryan"
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.bugMessage = ""
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.category = 95
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.description = "[center][b][size=6]Engine Fixes for Skyrim Special Edition[/size][/b][/center]\n<br />A WIP skse64 plugin for Skyrim Special Edition to fix bugs and patch issues.\n<br />\n<br />The majority of current work on this plugin (5.0+) is being done by Ryan, who deserves more credit than you can imagine :^)\n<br />\n<br />[size=4][b][center]Installation[/center][/b][/size]\n<br />[list]\n<br />[*]Install the Part 1 zip as you would any SKSE plugin, be that through your mod manager or directly to your data folder.\n<br />[/list]   ﻿The \"_preload.txt\" file is required to be in the directory if you want to use the memory manager patch, so if your Mod Manager warns you about it, make sure it gets added.[list]\n<br />[*]Install the Part 2 zip by extracting it to your Skyrim SE folder (the three dlls should be alongside SkyrimSE.exe, etc)\n<br />[/list]\n<br />\n<br />For configuration see the INI Settings below. The defaults should be stable for all users.\n<br />\n<br />[color=#ff7700][b]If you prefer SSE Fixes' version of the same fixes to this one, you can install them together and Engine Fixes will automatically disable its conflicting patches :)\n<br />[/b][/color]\n<br />\n<br />[b][size=4][center]INI Settings/Configuration[/center][/size][/b]\n<br />[b][size=3]EngineFixes[/size][/b]\n<br />[list]\n<br />[*]VerboseLogging (default: false) - Enables more verbose logging in the log file.\n<br />[*]CleanSKSECosaves (default: true) - Deletes SKSE cosaves with no matching save on launch.\n<br />[/list]\n<br />[b][size=3]Patches[/size][/b]\n<br />[list]\n<br />[*]DisableChargenPrecache (default: false) - \"Precache Killer\" function, duplicated from RaceMenu.\n<br />[*]EnableAchievementsWIthMods (default: true) - Enables achievements when you have mods installed.\n<br />[*]FormCaching (default: true) - Caches recently accessed forms. Similar to SSE Fixes.\n<br />[*]MaxStdio (default: true) - Raises max file handle limit from 512 to 2048, which should fix \"False Save Corruption\" bug in most cases.\n<br />[*]RegularQuicksaves (default: false) - Makes quick saves use the regular save handler.\n<br />[*]SaveAddedSoundCategories (default: true) - Saves volume settings for mod-added sound categories to a separate ini file. This functionality is the same as Audio Overhaul for Skyrim SE's dll, but they will not conflict.\n<br />[*]ScrollingDoesntSwitchPOV (default: false) - Makes it so using your scroll wheel won't swap between 1st/3rd person. See LE mod [url=https://www.nexusmods.com/skyrim/mods/95800]here[/url]﻿\n<br />[*]SleepWaitTime (default: false) - Lets you change the time it takes to sleep or wait. See LE mod [url=https://www.nexusmods.com/skyrim/mods/95796]here[/url]﻿\n<br />[*]SleepWaitTimeModifier (default: 0.3) - The in-game default time is 1.0; smaller values speed up the sleep/wait time, higher values slow it down\n<br />[*]TreeLODReferenceCaching (default: true) - Requires FormCaching. Fixes the very slow Tree LOD function that causes framerate drops most noticeable in Riften. Similar to SSE Fixes.\n<br />[*]WaterflowAnimation (default: true) - Decouples water flow animation from in-game timescale, so that decreasing/increasing your timescale doesnt mess with the water animation speed.\n<br />[*]WaterflowSpeed (default: 20.0) - Sets the waterflow speed for the animation patch. The default of 20 is equal to the default timescale, and is probably OK unless you want the water to appear slower/faster for some reason.\n<br />[/list]\n<br />[b][size=3]Fixes[/size][/b]\n<br />[list]\n<br />[*]ArcheryDownwardAiming (default: true) - Fix a bug where projectiles sometimes don't fire properly if you're aiming downward. Ported from Cobb's LE fix (see [url=https://www.nexusmods.com/skyrim/articles/52111]here[/url])\n<br />[*]AnimationLoadSignedCrash (default: true) - Fix an improper unsigned/signed conversion. Should allow loading more animations before a CTD. [b]If you use [url=https://www.nexusmods.com/skyrimspecialedition/mods/31146]Animation Limit Crash Fix SSE[/url] I suggest disabling this to use that mod's fix instead.[/b]\n<br />[*]BethesdaNetCrash (default: true) - Fixes a crash on startup caused by improper handling of non-ASCII characters in Bethesda.net HTTP response headers. \n<br />[*]BSLightingAmbientSpecular (default: true) - Fixes a bug where lighting template Directional Ambient Specular &amp; Fresnel Power are broken\n<br />[*]BSLightingShaderForceAlphaTest (default: true) - Forces alpha testing in the lighting shader. Fixes object LOD reflections on water.\n<br />[*]BSLightingShaderParallaxBug (default: true) - Fixes a bug with parallax lighting shaders\n<br />[*]CalendarSkipping (default: true) - Fixes the bug listed under Notes on [url=https://www.creationkit.com/index.php?title=GetCurrentGameTime_-_Utility]this page[/url]\n<br />[*]DoublePerkApply (default: true) - Fixes a bug where NPCs in the player's cell will have their perk effects applied twice on game load\n<br />[*]EquipShoutEventSpam (default: true) - Fixes a bug where the 'Equip Shout' script would fire a shout equipped event even if the equip failed.\n<br />[*]GetKeywordItemCount (default: false) - Fixes the condition function \"GetKeywordItemCount\" returning incorrect values in some situations.\n<br />[*]GHeapLeakDetectionCrash (default: true) - Fixes a crash where scaleform attempts to report a memory leak but the code doesnt exist in Skyrim\n<br />[*]LipSync (default: true) - Fixes the lip sync bug, same as LE bug fixes.\n<br />[*]MemoryAccessErrors (default: true) - Fixes a handful of out-of-bounds or use-after-free bugs. Required for experimental memory patches.\n<br />[*]MO5STypo (default: true) - Fixes a typo in the ARMA (Armor Addon) form loader preventing 1st person female alternate texture sets from being loaded.\n<br />[*]PerkFragmentIsRunning (default: true) - See [url=https://github.com/SniffleMan/MiscFixesSSE/blob/master/README.md]Misc Fixes SSE's readme[/url]﻿ for details.\n<br />[*]RemovedSpellBook (default: true) - See [url=https://github.com/SniffleMan/MiscFixesSSE/blob/master/README.md]Misc Fixes SSE's readme[/url]﻿ for details.\n<br />[*]SaveScreenshots (default: true) - Fixes save game screenshots being blank when TAA is disabled.\n<br />[*]SlowTimeCameraMovement (default: true) - Fixes camera movement being slow during slow time effects.\n<br />[*]TreeReflections (default: true) - Fixes tree LOD reflections. No conflict with ENB.\n<br />[*]UnequipAllCrash (default: true) - Fixes a crash when invoking \"Actor.UnequipAll\" on an actor without an AI process.\n<br />[*]VerticalLookSensitivity (default: true) - Makes vertical look sensitivity not tied to framerate.\n<br />[*]WeaponBlockScaling (default: true) - Fixes weapon blocking so it [url=https://en.uesp.net/wiki/Skyrim:Block#Skill_Usage]correctly scales off of the blocking actor's weapon[/url].\n<br />[/list][b]\n<br />[b][size=3]Warnings[/size][/b]\n<br />[/b][list]\n<br />[*]DupeAddonNodes (default: true)\n<br />[/list]\n<br />   ADDN (Addon Node) forms have a node index that must be unique across your entire load order. A warning box will pop up if you have a conflict in your load order, and the conflicting mods will be logged to your Engine Fixes log file. To resolve this, you need to contact the mod author(s) whose mods have conflicts, and one of them will have to change which node index they are using. Please be courteous :)\n<br />\n<br />[list]\n<br />[*]RefHandleLimit (default: true)\n<br />[*]RefrMainMenuLimit (default: 800000)\n<br />[*]RefrLoadGameLimit (default: 1000000)\n<br />[/list]\n<br />\n<br />   For detailed information on this engine limitation, see [url=https://www.reddit.com/r/skyrimmods/comments/ag4wm7/psa_the_reference_handle_cap_or_diagnosing_one_of/]my reddit post[/url]﻿.\n<br />\n<br />   This will pop up a warning box if your reference handle limit is above the configured warning levels at your main menu and whenever you load a save game.\n<br />\n<br />   Over the course of a 100+ hour game session, it is easily possible to build a large reference count in your save game, so the Main Menu limit is by default configured to 800k, which is 250k~ below the actual limit. If you don't intend to play long saves you can probably safely raise this warning limit.\n<br />\n<br />   The load game limit is set to 1 million (48k below the limit). If your save game is getting dangerously close to the limit it is possible to reduce the references stored in it using [url=https://www.nexusmods.com/skyrimspecialedition/mods/5031/]ReSaver[/url]﻿, but keep in mind this can cause things in your game world to disappear/respawn depending on your choices.\n<br />\n<br />   Currently, the only solution to this issue is to either remove non-master plugins with large reference counts from your load order, or turn them into master plugins.[b]\n<br />\n<br />[b][size=3]Experimental[/size][/b]\n<br />[/b][list]\n<br />[*]CellInit (default: true) - Fixes a rare crash where a form does not get converted from an id to a pointer.\n<br />[*]MemoryManager (default: false) - Disables built-in memory manager for the game's largest heaps. This is the famous \"Use OS Allocators\" patch \n<br />from LE.[*]UseTBBMalloc (default: true) - If you enable the memory manager patch, this will also replace standard malloc with tbbmalloc. This does nothing if the memory manager patch is disabled.\n<br />[*]SaveGameMaxSize (default: false) - Expands maximum save game size from 64 MB (uncompressed) to 128 MB. If you have a long playthrough and your game starts crashing when you try to save, this might fix it. Marked experimental because I can't test it myself, but is most likely safe.\n<br />[/list][b]\n<br />\n<br />[b][size=4][center]Credits[/center][/size][/b]\n<br />[/b]Nukem -  more stuff than I can mention\n<br />Sniffleman/Ryan - Misc Fixes + CommonLibSSE \n<br />meh321 - research into tree LOD function (SSE fixes), bugfixes LE &amp; port permissions\n<br />sheson - skse plugin preloader for LE alongside meh\n<br />himika - scatter table implementation from libskyrim (LE), plus tons of research function/variable names\n<br />kassent - useful information from the source code of various skse plugins \n<br />Kole6738 - cosave cleaner idea+code\n<br />LStewieAL - Things ported from [url=https://www.nexusmods.com/skyrim/users/2232669?tab=user+files&amp;BH=0]here[/url][b]\n<br />\n<br />[/b][center][b][size=4]Source &amp; Contributing[/size][/b][/center]\n<br />Source is available on [url=https://github.com/aers/EngineFixesSkyrim64]Github[/url]﻿. While I intend to look into some other known bugs, no promises. If you feel like tracking down a bug and theorizing a fix or even implementing one yourself, feel free to contribute; I'm willing to maintain it for future game updates.\n<br />\n<br />Preloader can be found [url=https://github.com/aers/skse64_plugin_preloader]here[/url]﻿.\n<br /> tbb.dll and tbbmalloc.dll are Intel's [url=https://www.threadingbuildingblocks.org/]Threaded Building Blocks[/url]﻿ library."
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.downloadGame = "skyrimse"
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.endorsed = "Endorsed"
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.fileId = 157938
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.fileMD5 = "b03366a5d439732a63687e8556ca0c2e"
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.fileName = "(Part 1) Engine Fixes-17230-5-4-1-1598558381.zip"
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.fileSize = 360887
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.fileType = "MAIN"
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.homepage = "https://www.nexusmods.com/skyrimspecialedition/mods/17230/"
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.installTime = "2020-12-03T16:07:26.573Z"
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.isPrimary = true
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.lastUpdateTime = 1612780840185
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.logicalFileName = "(Part 1) Engine Fixes"
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.modId = 17230
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.modName = "SSE Engine Fixes (skse64 plugin)"
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.modVersion = "5.4.1"
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.name = "(Part 1) Engine Fixes-17230-5-4-1-1598558381"
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.newestChangelog.content = "</br>"
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.newestChangelog.format = "html"
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.newestFileId = 181373
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.newestVersion = "5.6.0"
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.pictureUrl = "https://staticdelivery.nexusmods.com/mods/1704/images/17230/17230-1526195370-298143229.png"
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.shortDescription = "skse64 plugin to fix various issues with the Skyrim Special Edition engine."
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.source = "nexus"
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.uploadedTimestamp = 1598558381
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.version = "5.4.1"
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.fileOverrides = []
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.id = "(Part 1) Engine Fixes-17230-5-4-1-1598558381"
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.installationPath = "(Part 1) Engine Fixes-17230-5-4-1-1598558381"
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.state = "installed"
persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.type = ""
```

**Comment by TanninOne** [comment]
The mod has a changelog. It consists of a single linebreak:

> persistent.mods.skyrimse.(Part 1) Engine Fixes-17230-5-4-1-1598558381.attributes.newestChangelog.content = "\</br>"

**Comment by Pickysaurus** [comment]
I'm not sure which specific changelog it's trying to show, but it's not possible to have a changelog with just a linebreak on the site. 

![image](https://user-images.githubusercontent.com/31670524/115708970-8f3d0800-a368-11eb-90d0-8bec2db4573b.png)
